__tests__/app.test.js

```diff
@@ -127,10 +127,65 @@ describe("GET /api/articles", () => {
   });
   test("200: Articles are sorted by date in descending order by default", () => {
     return request(app)
-    .get("/api/articles")
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with the comments for the specified article_id", () => {
+    const expectedComments = [
+      {
+        body: "This is a bad article name",
+        votes: 1,
+        author: "butter_bridge",
+        article_id: 6,
+        created_at: "2020-10-11T15:23:00.000Z",
+        comment_id: 16,
+      },
+    ];
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toEqual(expectedComments);
+      });
+  });
```

Comment:
This is a really good test. I would recommend looking into the .toMatchObject matcher as an alternative which allows you to be a little more flexible with your assertions.